html/includes/common/availability-map.inc.php

```diff
@@ -93,6 +101,18 @@
                 $host_down_count++;
             }
 
+            if ($config['force_ip_to_sysname'] === true) {
```

Comment:
I'm not really sure what all of this if/elseif/else statement is doing but surely you can just do:

`$deviceDisplay = ip_to_sysname($device['hostname']);`